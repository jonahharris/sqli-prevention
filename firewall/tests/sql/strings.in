>>> Sada testovacich SQL dotazu: retezce
SELECT 'MariaDB\'s new features';
SELECT 'I''m here';
SELECT """What?""";
SELECT 'Hellooo\"';
SELECT 'Tab \t tab';
SELECT 'New \n line';
SELECT 'Null \0 char';
SELECT 'Backspace \b char';
SELECT 'New \n\r line with carriage return';
SELECT 'Control+Z \Z here';
SELECT 'Backslash \\ char';
SELECT 'Percent \% char';
SELECT 'Underline \_ char';
