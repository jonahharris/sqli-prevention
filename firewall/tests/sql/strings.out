>>> Sada testovacich SQL dotazu: retezce
STRING '***';SELECTNODATA 0 1;STMT;1;0;
STRING '***';SELECTNODATA 0 1;STMT;1;0;
STRING '***';SELECTNODATA 0 1;STMT;1;0;
STRING '***';SELECTNODATA 0 1;STMT;1;0;
STRING '***';SELECTNODATA 0 1;STMT;1;0;
STRING '***';SELECTNODATA 0 1;STMT;1;0;
STRING '***';SELECTNODATA 0 1;STMT;1;0;
STRING '***';SELECTNODATA 0 1;STMT;1;0;
STRING '***';SELECTNODATA 0 1;STMT;1;0;
STRING '***';SELECTNODATA 0 1;STMT;1;0;
STRING '***';SELECTNODATA 0 1;STMT;1;0;
STRING '***';SELECTNODATA 0 1;STMT;1;0;
STRING '***';SELECTNODATA 0 1;STMT;1;0;
